metadata.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "chocolatey-chocolatey",
-  "version": "1.2.1",
+  "name": "puppetlabs-chocolatey",
+  "version": "0.0.1",
```

Comment:
Because it needs to changed to something other than 1.2.1.  Given there is no definite version number to use yet, I used the lowest number I could.